lib/Utils/Backends.pm

```diff
@@ -55,6 +56,12 @@ our %EXPORT_TAGS = (
     BACKEND  => (BACKEND)
 );
 
+sub set_sshserial_serialdev {
```

Comment:
What about `set_sshserial_dev` ? To make it shorter.

Reply:
Good point, changed. Thanks for a review.

Reply:
I'd go for `set_ssh_serial_dev` Just for consistency but it's not really important

Reply:
Isn't all this handled by select_serial_terminal ? Why we needanother `set_sshserial_serialdev` and keep `use_ssh_serial_console` ?!